harness/determined/common/api/bindings.py

```diff
@@ -5037,22 +5037,26 @@ def __init__(
         self,
         *,
         agentUserGroup: "typing.Optional[v1AgentUserGroup]" = None,
+        checkpointStorageConfig: "typing.Optional[typing.Dict[str, typing.Any]]" = None,
         name: "typing.Optional[str]" = None,
     ):
         self.name = name
         self.agentUserGroup = agentUserGroup
+        self.checkpointStorageConfig = checkpointStorageConfig
 
     @classmethod
     def from_json(cls, obj: Json) -> "v1PatchWorkspace":
         return cls(
             name=obj.get("name", None),
             agentUserGroup=v1AgentUserGroup.from_json(obj["agentUserGroup"]) if obj.get("agentUserGroup", None) is not None else None,
+            checkpointStorageConfig=obj.get("checkpointStorageConfig", None),
         )
 
     def to_json(self) -> typing.Any:
         return {
             "name": self.name if self.name is not None else None,
             "agentUserGroup": self.agentUserGroup.to_json() if self.agentUserGroup is not None else None,
+            "checkpointStorageConfig": self.checkpointStorageConfig if self.checkpointStorageConfig is not None else None,
```

Comment:
not relevant to this PR, but this auto-generated line is kinda useless as it is equivalent to `self.checkpointStorageConfig`.